README.md

```
@@ -124,8 +109,7 @@ docker compose logs -f qdrant
 docker compose up --build backend
 ```
 
-### Access Neo4j Browser
+### Access Qdrant Dashboard
 
-1. Go to http://localhost:7474
-2. Connect with: `neo4j` / `sanctum_dev_password`
-3. Query: `MATCH (n) RETURN n`
+1. Go to http://localhost:6333/dashboard
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)